catalog/src/main/kotlin/com/adevinta/spark/catalog/configurator/Navigation.kt

```diff
@@ -60,5 +58,6 @@ internal fun NavGraphBuilder.navGraph(
     }
 }
 
-internal const val ConfiguratorRoute = "configurator"
-internal const val ComponentIdArgName = "componentId"
+private fun NavController.navigateToConfiguratorShowcase(id: Int) {
+    navigate(route = ConfiguratorShowcase(componentId = id))
+}
```

Comment:
Do we need to extract this single usage into a function?
In other parts of the code, you don't extract them.
We should choose one consistent way of doing it. Personnaly, I would inline these at the call site directly.

Reply:
Fixed in https://github.com/adevinta/spark-android/pull/1401/commits/b231b1cc07ea384f7c81c9a628d7d93e900eb9a1